backend/src/main/resources/sql/schema.sql

```diff
@@ -0,0 +1,41 @@
+DROP TABLE IF EXISTS CARD, ACTION_LOG;
+CREATE TABLE CARD (
+    id int AUTO_INCREMENT PRIMARY KEY,
+    title varchar(50) NOT NULL,
+    content varchar(500),
+    author_system varchar(20) NOT NULL,
+    status ENUM('to_do', 'in_progress', 'done'),
+    deleted tinyint(1) DEFAULT 0,
+    order_index int NOT NULL
```

Comment:
이런 부분도 인덴트가 잘 나눠져있으면 가독성이 훨씬 좋아질겁니다. 예시는 참고만 해주세요~
```suggestion
    id            int AUTO_INCREMENT PRIMARY KEY,
    title         varchar(50) NOT NULL,
    content       varchar(500),
    author_system varchar(20) NOT NULL,
    status        ENUM('to_do', 'in_progress', 'done'),
    deleted       tinyint(1) DEFAULT 0,
    order_index   int         NOT NULL
```

Reply:
가독성 개선 팁 감사합니다 👍